README.md

```diff
@@ -0,0 +1,60 @@
+# LinkedList
+
```

Comment:
Add CI and doc badge

Reply:
Fixed  now linked_list are ready to pushed crates-io